cdap-api/src/main/java/co/cask/cdap/api/artifact/ArtifactId.java

```diff
@@ -49,11 +49,7 @@ public ArtifactScope getScope() {
 
   @Override
   public String toString() {
-    return "ArtifactId{" +
-      "name='" + name + '\'' +
-      ", version=" + version +
-      ", scope='" + scope + '\'' +
-      '}';
+    return String.format("%s-%s-%s", scope, name, version);
```

Comment:
May be I will add a method to generate the file name of the Jar in ArtifactId and leave the toString as is.